state-synchronizer/src/executor_proxy.rs

```diff
@@ -146,11 +146,14 @@ impl ExecutorProxyTrait for ExecutorProxy {
         intermediate_end_of_epoch_li: Option<LedgerInfoWithSignatures>,
         _synced_trees: &mut ExecutedTrees,
     ) -> Result<()> {
+        // track chunk execution time
+        let execute_start_time = Instant::now();
```

Comment:
timer doesn't work?

Reply:
this is `DurationHistogram`, which is just a wrapper around `Histogram`but doesn't have start_timer support

Reply:
actually i can just change this to `Histogram`, I don't see any real advantage to `DurationHistogram`